submitted_models/coro_allie_sensor_config_1/launch/description.launch

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0"?>
+<launch>
+  <arg name="name" doc="Name of Vehicle"/>
+  <param name="$(arg name)/robot_description" command="$(find xacro)/xacro '$(find coro_allie_sensor_config_1)/urdf/model.xacro' name:=$(arg name)"/>
```

Comment:
The `model.xacro` file is missing. Instead it looks like you have a `model.urdf` file. I think there might be some transform issues in the urdf. For example I see errors in rviz when visualizing the `RobotModel`. Can you take a look?

Reply:
I fixed the launch file to point at the .urdf.  But, the urdf is probably not correct based on your observations.  I used https://github.com/andreasBihlmaier/pysdf to convert the sdf to a urdf as I couldn't find a sdf to xacro translator.

Is there a better way to create the .xacro or does it need to be hand edited based on the .sdf?
Is there a process to test the resulting urdf/xacro?

Reply:
@knoedler Did you try following the instructions on the [Model Migration Guide](https://github.com/osrf/subt/wiki/Model-Migration-Guide#rviz-visualization)?

Reply:
Thank you for the reference.  No, I had not looked in the Gazebo 9 Migration guide, it hadn't come up in my google searches either.  That is the set of steps I was looking for.  Since the urdf / xacro is optional I have disabled it for now and added a feature improvement task to my list.